tests/helpers/mocks.js

```diff
@@ -46,6 +46,7 @@ export const createActividadMock = (fields) => EmberObject.extend({
 }).create()
 
 export const actividadMock = createActividadMock()
+export const actividadConExpectativaMock = createActividadMock({ expectations: { conditionalAlternative: true }, grupo: undefined })
```

Comment:
No usemos más "actividad", usemos "challenge" , y usemos inglés, siempre inglés.

```suggestion
export const challengeWithExpectationsMock = createActividadMock({ expectations: { conditionalAlternative: true }, grupo: undefined })
```